settings/common.py

```diff
@@ -11,10 +11,12 @@
 """
 
 import os
-
+import sys
 # Build paths inside the project like this: os.path.join(BASE_DIR, ...)
 BASE_DIR = os.path.dirname(os.path.dirname(os.path.abspath(__file__)))
+APPS_DIR = os.path.join(BASE_DIR, 'apps')
 
+sys.path.append(APPS_DIR)
```

Comment:
If i remove apps from system path, and change all import to be of the form `from apps.X.models import Y` I still get an error 'No module named X.models`

Any suggestions to get rid of this?